_posts/2021-03-31-grant-announcements.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Announcing Brink's first round of developer grants"
+permalink: /blog/2021/03/31/grantees/
+layout: post
+author: brink
+name: "brink"
+alt: Brink announces first round of developer grants
+category: "News"
+description: Brink announces first round of developer grants for Bitcoin Core, Bitcoin Dev Kit, and a FROST Implementation.
+---
+
+Brink is proud to announce the recipients of its first round of open source developer grants.
+After an open grant application process, the [Brink board of directors][brink board] has selected three
+developers who have demonstrated continued commitment to open source Bitcoin protocol
+development. Their work spans fundamental cryptographic engineering, application tooling,
+and base layer development and maintenance. Brink supports each of these developers to
+work **full time** on open source development. The three grant recipients are:
+
+## Jesse Posner
+
+<img src="/assets/images/jesse-posner.jpg" alt="Picture of Jesse Posner" style="float:left;border:1px solid
+darkgray;margin-right:25px;width:150px;" />
+
+Jesse’s funding supports his work on adaptor signatures and an implementation of the
+[FROST threshold signature scheme][frost]. These technologies bring additional security, privacy
+and functionality to multisignature and second layer protocols.
+
+[@jesseposner][jesse twitter]
+
+_Grant made possible by very generous financial support from [Nexo][nexo]._
+<br style="clear:both" />
+
+## Alekos Filini
+
+<img src="/assets/images/alekos-filini.jpg" alt="Picture of Alekos Filini" style="float:left;border:1px solid
+darkgray;margin-right:25px;width:150px;" />
+Alekos's funding supports his work as maintainer of the [Bitcoin Developer Kit][bdk]. BDK is a
+lightweight and modular collection of tools for use by wallets and other Bitcoin applications.
```

Comment:
Nit: "collection of tools" sounds to me like a collection of binaries.  Maybe call it a "library" or a "toolkit"?

Reply:
This was taken (slightly modified) from https://bitcoindevkit.org/:

> The Bitcoin Dev Kit (BDK) project [...] aims to build a collection of tools and libraries [..] All BDK components are designed to be lightweight and modular.